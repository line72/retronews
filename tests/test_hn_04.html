<p>Lorem ipsum dolor sit amet, pro eu soleat civibus. Mel quas sensibus</p><p>[0]: <a>https://long.long.long.long.long.long.long.long.long.long.long.example.com</a></p><p>[1] - <a>https://long.long.long.long.long.long.long.long.long.long.long.example.com</a></p><p>[2] <a>https://long.long.long.long.long.long.long.long.long.long.long.example.com</a></p>
