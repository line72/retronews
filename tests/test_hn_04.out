Lorem ipsum dolor sit amet, pro eu soleat civibus. Mel quas sensibus

[0]: https://long.long.long.long.long.long.long.long.long.long.long.example.com

[1] - https://long.long.long.long.long.long.long.long.long.long.long.example.com

[2] https://long.long.long.long.long.long.long.long.long.long.long.example.com